src/commands/MigrationGenerateCommand.ts

```diff
@@ -196,6 +196,7 @@ export class MigrationGenerateCommand implements yargs.CommandModule {
                         )} has been generated successfully.`,
                     ),
                 )
+                process.exit(0)
```

Comment:
It's default behavior. Process will exist with code 0 once execution is finished anyway. Why do we need to add this line?

Reply:
I don't know about others here, but my TypeORM commands never exit (using version 0.3.7)

Reply:
we need to understand why. Force exist isn't an option if there are some resources that got busy. We may "hide" the actual bug this way.

Reply:
Haven't looked that far yet why it is behaving this way. It just gets stuck after execution. However, TypeORM CLI did properly exit for version 0.2.x.

Reply:
Migration generate command was never existing for me. even before v0.3.x. (using mysql)

Reply:
Maybe same resolution was applied there. So better to find a valid reason this time.

Reply:
I too have the same problem with exit in nestjs if i using `app. enableShutdownHooks()`